fecgov/fecfile-web-api
# Created F3X API view and integrate validation

#7 
Adds F3X crud endpoints and integrates fecfile-validate into the new serializer.
Removes staticfiles
Cleans up log settings and adds more logging to apps

To test you can 
POST to `/api/v1/f3x-summaries/ `with a valid f3x in json, like 
```
"form_type": "F3XN",
"filer_committee_id_number": "C00000000",
"treasurer_last_name": "Lastname",
"treasurer_first_name": "Firstname",
"date_signed": "20040729",
"created": "2022-02-09T00:00:00.000Z",
"updated": "2022-02-09T00:00:00.000Z"
```
PUT to `/api/v1/f3x-summaries/<id>/` with valid f3x in json to update
DELETE to `/api/v1/f3x-summaries/<id>/` to soft delete
GET to `/api/v1/f3x-summaries/<id>/` to get an f3x

see validation errors (note the 400 response):
![image](https://user-images.githubusercontent.com/97105825/160939153-e5b19bf0-e29a-4807-960f-88a081e5a400.png)
